File: app/src/main/java/com/example/bmicalculator/QuizItem.java
package com.example.bmicalculator;

public class QuizItem {

    private final String question;

    public String getQuestion() {
        return question;
    }

    public String getRightAnswer() {
        return rightAnswer;
    }

    private final String rightAnswer;

    public QuizItem(String question, String rightAnswer) {
        this.question = question;
        this.rightAnswer = rightAnswer;
    }
}

File: app/src/main/java/com/example/bmicalculator/QuizItem.kt
package com.example.bmicalculator

class QuizItem(val question: String, val answers: List<Pair<String, Boolean>>)


val quiz = listOf(
        QuizItem("Polska", listOf(
                Pair("Londyn", false),
                Pair("Berlin", false),
                Pair("Moskwa", false),
                Pair("Warszawa", true)
        )),
        QuizItem("UK", listOf(
                Pair("Londyn", true),
                Pair("Berlin", false),
                Pair("Moskwa", false),
                Pair("Warszawa", false)
        )),
        QuizItem("Niemcy", listOf(
                Pair("Londyn", false),
                Pair("Berlin", true),
                Pair("Moskwa", false),
                Pair("Warszawa", false)
        ))
)